submarine-server/server-submitter/submitter-yarn/src/main/java/org/apache/submarine/runtimes/tony/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
Need `package-info.java`?

Reply:
This is required by the Apache Hadoop code check.  We seem to don't have this. It's ok to keep it